docs/contributing/contributing-to-airbyte/README.md

```diff
@@ -4,45 +4,92 @@ description: 'We love contributions to Airbyte, big or small.'
 
 # Contributing to Airbyte
 
-We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get input from our community.
+_First_: if you feel insecure about how to start contributing, feel free to ask us on [Slack](https://slack.airbyte.io) in the \#dev or even \#general channel. You can also just go ahead with your contribution and we'll give you feedback. Don't worry,  the worst that can happen is that you'll be politely asked to change something! We appreciate any contributions, and we don't want a wall of rules to stand in the way of that.
```

Comment:
I wouldn't use the word insecure, it's a bit too strong "if you don't know where to start" ?